man/systemd-firstboot.xml

```diff
@@ -221,6 +221,15 @@
         <option>--root=</option>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--overwrite</option></term>
```

Comment:
hmmm, we have a `--force` switch at various places, but no `--overwrite` so far. I wonder if it wouldn't be better to reuse the `--force` name for this, given that this would be closer to our usual nomenclature for these things?